go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/terraform-providers/terraform-provider-mso
 go 1.13
 
 require (
-	github.com/ciscoecosystem/mso-go-client v1.11.2
```

Comment:
Is this the correct version? thought mac attribute was added in 1.15.0?

Reply:
v1.17.0 includes the mac attribute

Reply:
but also other features like vrf route-leaking, will this not cause issue for compile?

Reply:
You mean while testing?

Reply:
Or any other build of master, could be during testing or people building of master themself. If there is function used with a go-client change that is not the same as the arguments passed in the terraform provider it will not compile. Guess this is the downside of the dependency and it could also happen if you do 1.15.0 if the changes in 1.14.0 for instance are causing it. So never mind we can just accept the latest I guess.